src/lib/vocab/space/spaceRepo.ts

```diff
@@ -80,4 +80,33 @@ export const spaceRepo = (db: Database) => ({
 		// console.log('[db] created default spaces', community_id, spaces);
 		return {ok: true, value: spaces};
 	},
+	deleteById: async (
+		space_id: number,
+	): Promise<Result<{value: any[]}, {type: 'deletion_error'} & ErrorResponse>> => {
+		const data_cs = await db.sql<any[]>`
+      DELETE FROM community_spaces WHERE ${space_id}=space_id
+    `;
+
+		if (data_cs.count != 1) {
```

Comment:
To support multiples i think this should be:
```suggestion
		if (data_cs.count === 0) {
```

Reply:
This can only delete 1 space at a time though? The input param is a single number, not an array.

Reply:
It's the `community_spaces` table, so it'll error if more than one community has the space.